android/src/main/java/com/apparence/camerawesome/CameraPicture.java

```diff
@@ -144,6 +202,13 @@ public void dispose() {
             pictureImageReader.close();
             pictureImageReader = null;
         }
+        if (recorder != null) {
+            recorder.release();
```

Comment:
Should recorder become null? 
```suggestion
            recorder.release();
            recorder = null;
```